common/shlibs

```diff
@@ -3525,7 +3525,7 @@ libcsfml-window.so.2.5 CSFML-2.5_1
 libcsfml-system.so.2.5 CSFML-2.5_1
 libcsfml-network.so.2.5 CSFML-2.5_1
 libtokyocabinet.so.9 tokyocabinet-1.4.48_1
-libdav1d.so.6 libdav1d6-1.0.0_1
+libdav1d.so.7 libdav1d6-1.3.0_1
```

Comment:
Why does the subpackage have a 6 at the end of the name? Maybe this should be just `libdav1d`.